constructor-manager-api/src/constructor_manager_api/utils/conda.py

```diff
@@ -0,0 +1,47 @@
+"""Conda utilities."""
+
+import sys
+from pathlib import Path
+from typing import Optional
+
+
+def get_base_prefix() -> Path:
+    """Get base conda prefix.
+
+    Returns
+    -------
+    pathlib.Path
+        Base conda prefix.
+    """
+    current = Path(sys.prefix)
+    if (current / "envs").exists() and (current / "envs").is_dir():
+        return current
+
+    if current.parent.name == "envs" and current.parent.is_dir():
```

Comment:
Should this also check if the path exists (as above)? It might be enough for each to just check `is_dir()` as I don't think it just returns `False` if the path doesn't exist.

Reply:
Good point!